### DISCUSSION TITLE
Plotting Error, Stuck at Phase -2, Some temp files are created of 0Kb

### DISCUSSION THREAD

**Discussion by bhimanbaghel** [Chia-Network/chia-blockchain#5551] [Support]
Error Message:
Only read 0 of 1048576 bytes at offset 5682179154 from "F:\\ChiaTemp\\plot-k33-2021-05-17-02-35-815e802d08911be3c02426907d3b8024cdafdc2eeb13b3836680f7604f4f6c62.plot.table2.tmp"with length 44564351718. Error 1. Retrying in five minutes.

Was plotting a single k-33 using an external HDD of 4.5TB. So space doesn't seem an issue.

**Is there a way to proceed with replotting?**
Thanks

Peek at temp dir:
![temp_plot](https://user-images.githubusercontent.com/13465113/118873477-709a4480-b907-11eb-8279-eb50703c4e90.PNG)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
